Add an item-collection-tutorial guide. (#310)
cerner/terra-ui: src/terra-dev-site/doc/clinical-item-collection/clinicalItemCollection.1.doc.mdx

> import { Notice } from '@cerner/terra-docs';

Reviewer: shouldn't this deprecated notice be directly embedded in the guide?

> 
> <Notice variant="deprecation" ariaLevel="3">
> 
>   This component has been deprecated and should no longer be used. Refer to [Recreating Item Collection](/guides/terra-ui/component-guides/recreating-item-collection).
>   
> </Notice>

Reviewer 2: ```suggestion
# Terra Clinical Item Collection

<Notice variant="deprecation" ariaLevel="3">

  This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection was too limited in API to meet the varied needs of all consumers and instead are now recommending teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/recreating-item-collection) for replacement recommendations.
  
</Notice>
```

import { Notice } from '@cerner/terra-docs';

# Terra Clinical Item Collection

<Notice variant="deprecation" ariaLevel="3">

>   This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection API was too limited to meet the varied needs of all consumers. Instead, we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/recreating-item-collection) for replacement recommendations.

Reviewer 3: I think this should be:

```suggestion
  This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection API was too limited to meet the varied needs of all consumers. Instead, we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/building-an-item-collection) for replacement recommendations.
```

>   This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection API was too limited to meet the varied needs of all consumers. Instead, we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/building-an-item-collection) for replacement recommendations.

Reviewer 3: ```suggestion
  This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection API was too limited to meet the varied needs of all consumers. Instead, we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Building an Item Collection Guide](/guides/terra-ui/component-guides/building-an-item-collection) for replacement recommendations.
```

>   This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection was too limited in API to meet the varied needs of all consumers and instead we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/recreating-item-collection) for replacement recommendations.

Reviewer: ```suggestion
  This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection API was too limited to meet the varied needs of all consumers. Instead, we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/recreating-item-collection) for replacement recommendations.
```

>   This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection was too limited in API to meet the varied needs of all consumers and instead are now recommending teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/recreating-item-collection) for replacement recommendations.

Reviewer 4: ```suggestion
  This component has been deprecated in favor of encouraging teams to create their own component. We found that the item collection was too limited in API to meet the varied needs of all consumers and instead we now recommend teams use terra components to create their own flexible list/table component. Please refer to the [Recreating Item Collection Guide](/guides/terra-ui/component-guides/recreating-item-collection) for replacement recommendations.
```
